faunadb-java/src/test/resources/logback.xml

```diff
@@ -7,9 +7,6 @@
         </encoder>
     </appender>
 
-    <logger name="com.ning.http.client.providers.netty.NettyAsyncHttpProvider" level="off"/>
```

Comment:
is there an alternative logger here that we need to squelch? If I remember correctly, it was super spammy with the default log levels.